governance_artifacts/oscal/generate_annex_iv_dossier.py

```
@@ -0,0 +1,373 @@
+
+def _run_check(command: str | None) -> bool | None:
+    """Run a control's backing assurance command; True/False, or None if no
+    runnable command (organisational evidence)."""
+    if not command:
+        return None
+    proc = subprocess.run(command, cwd=REPO_ROOT, shell=True,
+                          capture_output=True, text=True)
+    return proc.returncode == 0
+
+
+def build_dossier(verify_evidence: bool = True) -> dict:
```

Pylint (4.0.6)

[refactor] Too many local variables (19/15)

(R0914)